evm/src/cpu/columns.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
nit: let's add a TODO (empty is fine) to make sure we don't forget it, and maybe scope it more narrowly if it's not too many items?

Reply:
Hm it's all six of the functions, so I thought it was better to just do it throughout the file